site/content/user-guide/jmeter_load_tests.md

```diff
@@ -0,0 +1,34 @@
++++
+title = "JMeter Load Tests"
+weight = 300
```

Comment:
300 is the (starting) weight for the next sub menu (`Components`) ... so, maybe something between 290 and 300 if you want it to appear as the last page in the `Developer Guide` menu ...